src/main/cpp/auton/PrimitiveParser.cpp

```diff
@@ -190,6 +232,30 @@ PrimitiveParamsVector PrimitiveParser::ParseXML(string fulldirfile)
                             {
                                 pipelineMode = DragonCamera::PIPELINE::COLOR_THRESHOLD;
                             }
+                            else if (strcmp(attr.name(), "climberOption") == 0)
+                            {
+                                auto itr = xmlStringToCLIMBERSTATE_NAMESEnumMap.find(attr.value());
```

Comment:
Add check to make sure the mechanism exists before setting this

Reply:
Mr. Wits could you please help me with this part? I tried to create a bool variable in ClimberManagerGen/noteManagerGen and set it to true on their respective create methods, then seeing if that was true in PrimitiveParser but it popped up an error saying the variable was not a member of the class
Please helpppppp

Reply:
Yes; I think we may have talked about this, but if not, ask me when you are working on it.